docs/manual/publish.md

```diff
@@ -42,7 +42,25 @@ For example,
 <iframe src="http://< ip-address >:< port >/#/notebook/2B3QSZTKR/paragraph/...?asIframe" height="" width="" ></iframe>
 ```
 
+Don't forget to also add below lines in your page for loading the Twitter Bootstrap themes and CSS used for Zeppelin notebook style, or your paragraph will be shown ugly.
+
+```
+<head>
+     ....
+    <link rel="stylesheet" href="zeppelin-web/bower_components/bootstrap/dist/css/bootstrap.css" />
+    <link rel="stylesheet" href="zeppelin-web/fonts/font-awesome.min.css">
+    ....
+</head>
```

Comment:
I'm not sure it is required, while paragraph is exported as an iframe. Can some clarify it?